db/migrations/migrations_test.go

```diff
@@ -256,6 +255,109 @@ func (s *MigrationTestSuite) TestBCDAMigration() {
 				}
 			},
 		},
+		{
+			"Set ACO Termination Details Default",
+			func(t *testing.T) {
+				// create ACO that has been blacklisted by using createTestRow since "Blacklisted"
+				// is no longer an models.ACO property.
+				cmsID := testUtils.RandomHexID()[0:4]
+				acoUUID := uuid.NewUUID()
+				aco := map[string]interface{}{
+					"uuid":        acoUUID,
+					"cms_id":      cmsID,
+					"name":        "Blacklisted ACO",
+					"blacklisted": true,
+				}
+				createTestRow(t, db, "acos", aco)
+
+				defer postgrestest.DeleteACO(t, db, acoUUID)
+
+				migrator.runMigration(t, "11")
+				assertColumnExists(t, true, db, "acos", "termination_details")
+
+				sb := sqlFlavor.NewSelectBuilder()
+				sb.Select("blacklisted", "termination_details").
+					From("acos").
+					Where(
+						sb.Equal("blacklisted", true),
+					)
+
+				query, args := sb.Build()
+				rows, err := db.Query(query, args...)
+				assert.NoError(t, err)
+				defer rows.Close()
+
+				for rows.Next() {
+					var blacklisted bool
+					var termination_details string
+					assert.NoError(t, rows.Scan(&blacklisted, &termination_details))
+					assert.NotEqual(t, nullValue, termination_details,
+						"\"termination_details\" value is not supposed to be null.  Actual value is %s", termination_details,
+					)
+				}
+			},
+		},
+		{
+			"Remove ACO Blacklisted Column",
+			func(t *testing.T) {
+				assertColumnExists(t, true, db, "acos", "blacklisted")
+				migrator.runMigration(t, "12")
+				assertColumnExists(t, false, db, "acos", "blacklisted")
+			},
+		},
+		// down migrations tests begin here with test number - 1
+		{
+			"Add ACO Blacklisted Column",
+			func(t *testing.T) {
+				// create ACO that has been terminated with termination details.
+				cmsID := testUtils.RandomHexID()[0:4]
+				aco := models.ACO{
+					UUID:  uuid.NewUUID(),
+					CMSID: &cmsID,
+					Name:  "Blacklisted ACO",
+					TerminationDetails: &models.Termination{
+						TerminationDate: time.Date(2020, time.December, 31, 23, 59, 59, 0, time.Local),
+						CutoffDate:      time.Date(2020, time.December, 31, 23, 59, 59, 0, time.Local),
+						BlacklistType:   models.Involuntary,
+					}}
+
+				defer postgrestest.DeleteACO(t, db, aco.UUID)
+
+				postgrestest.CreateACO(t, db, aco)
+				assertColumnExists(t, false, db, "acos", "blacklisted")
+				migrator.runMigration(t, "11")
+				assertColumnExists(t, true, db, "acos", "blacklisted")
+
+				sb := sqlFlavor.NewSelectBuilder()
+				sb.Select("termination_details", "blacklisted").
+					From("acos").
+					Where(
+						sb.IsNotNull("termination_details"),
+					)
+
+				query, args := sb.Build()
+				rows, err := db.Query(query, args...)
+				assert.NoError(t, err)
+				defer rows.Close()
+
+				for rows.Next() {
```

Comment:
Same comment as above, can we ensure that we actually read out rows?

Reply:
Fixed